subsys/partition_manager/Kconfig

```diff
@@ -182,11 +182,26 @@ config PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY
 	# option is propagated to the MCUboot child image automatically.
 	depends on PM_EXTERNAL_FLASH_ENABLED || MCUBOOT
 	default y if BOOTLOADER_MCUBOOT
+	imply PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY_1
```

Comment:
Would like @tejlmand to comment on this from a sysbuild perspective, how can this be used from sysbuild since these do not apply

Reply:
I don't know how sysbuild handles putting things in external flash to be honest. I'm a bit out of the loop on the whole sysbuild system also :(

Reply:
@sigvartmh @tejlmand @nordicjm Can you find an agreement here?